misc/talon_helpers.talon

```diff
@@ -57,3 +57,7 @@ talon dump context:
 ^talon copy active app$:
     result = user.talon_get_active_application_info()
     clip.set_text(result)
+
+^report talon bug$: 'https://github.com/talonvoice/talon/issues'
+^report knausj bug$:'https://github.com/knausj85/knausj_talon/issues'
```

Comment:
We can use `user.open_url` here to go directly to the website.

Also, the naming convention suggests this should be `talon report bug` (or `talon bug report`, or both), although I'm fine with having both synonyms. I would suggest that we just have one command, which points to `https://github.com/knausj85/knausj_talon/issues`, since most users may not be able to tell what's a knausj bug and what's a talon bug.